### ISSUE TITLE
Can we change password length?

### ISSUE THREAD

**Issue by nfischer** [freitagbr/diceware#4]
5 words is good, but users may want different lengths. Any API to do this?

It would be cool to display entropy information after the password is generated (e.g. "your 5-word password has x bits of entropy (moderately secure)").

**Comment by freitagbr** [comment]
The `-n N` flag lets you configure the number of words generated, but not the total length. I could add this.

Yeah, displaying the bits of entropy would be cool. Also, if the generated password does not meet a certain entropy threshold, it could be regenerated.

**Comment by nfischer** [comment]
Yeah, I just meant number of words generated. Entropy in this scheme should only depend on number of words (each word is 12.9 bits of entropy, I believe).
